packages/core/integration-tests/test/integration/shared-bundle-internalization/large.mjs

```diff
@@ -0,0 +1,6 @@
+import "./async.mjs";
+import mod from "./imports-async.mjs";
+
+let text = "lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor lorem ipsum dolor";
```

Comment:
One day, swc might decide to optimize this variable away, then it wouldn't be a shared bundle anymore.

Reply:
Yeah, I had the same concern and had a console.log here in an earlier iteration. Maybe we should have a global function that's a noop or something in the test env?